charts/rasa-x/templates/rasa-services.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
   labels:
     {{ include "rasa-x.labels" $ | nindent 4 }}
     app.kubernetes.io/component: {{ .serviceName }}
+{{- with .Values.rasa.service.annotations }}
```

Comment:
The rasa service is a bit tricky
```suggestion
{{- with $.Values.rasa.service.annotations }}
```

Reply:
Ha, I'd just figured this out 🙈  thanks!